stacks/api/bucket/src/crud.ts

```diff
@@ -87,6 +87,15 @@ export async function findDocuments<T>(
     params.req.res.header("Content-language", locale.best || locale.fallback);
   }
 
+  // matching for rules and filters
+  pipeline.push({
+    $set: {
+      _id: {
+        $toString: "$_id"
+      }
+    }
+  });
+
```

Comment:
Keep in mind that this might cause `findOne` to do a `COLL_SCAN` because by doing this you are discarding the indexes on _id.